src/main/java/com/example/javaee_jpa_hibernate/model/InvoiceBuilder.java

```diff
@@ -12,7 +12,7 @@
 import java.util.List;
 
 public class InvoiceBuilder {
-    private Long id;
+//    private Long id;
```

Comment:
In the Invoice class you've mark `id` as a generated field, so why you're setting it manually?